stackoverflow-dark.user.css

```
@@ -1422,7 +1423,7 @@ regexp("^https?:\/\/((?!area51)\w+\.)*?stackexchange\.com.*") {
   }
   /* Assign */
   #answers .answer.accepted-answer {
-    background: rgba(0, 40, 16, .5) !important;
+    background: rgba(/*[[accepted-color-rgb]]*/, .5) !important;
     border: 1px solid #13552f !important;
```

Also need to come up with a border custom color I would say 

```suggestion
    border: 1px solid rgba(/*[[accepted-color-rgb]]*/, 1) !important;
```

so that way border always brighter than the bg.

---

Also we need to find a way to deal with this https://github.com/StylishThemes/StackOverflow-Dark/pull/194/files#diff-461826e838232dd067ac76c1f6ee5ee7R1410-R1422 see issue #187 for why this is necessary.

Thats part and parcel of this new option, If people choose anything other than green the animation will be green so we need to also solve all those properties before this can be merged also =).

Ive no idea about that at this time. Will need to play around when I have time.

Suggestions welcome though.